tests/run_tests.py

```diff
@@ -0,0 +1,43 @@
+import os
+from io import StringIO
+import difflib
+
+from rockstarpy.rockstar import convert_code
+
+
+def check_files_identical(expected, actual):
+    diff = difflib.unified_diff(
+                expected,
+                actual,
+                fromfile='expected',
+                tofile='actual',
+    )
+    diff = list(diff)
+    if len(diff):
+        for d in diff:
+            print(diff)
+        assert False, "There are differences"
+
+def main():
+    files = os.listdir('.')
+    rock_files = filter(lambda f: f.split(".")[-1] in ['rock','rockstar','lyrics'] , files)
+    py_files = set(filter(lambda f: '.py' == f[-3:], files))
+    for rock_file in rock_files:
+        file_name = rock_file[:-5]
+        py_file = file_name + ".py"
+        assert py_file in py_files, "Did not create a corrosponding expected output for " + file_name + ".rock"
+    
```

Comment:
Good, there is some code that can be improved:

line 24: `py_files = set(filter(lambda f: f.endswith('.py'), files))`
line 26: `file_name = rock_file.rstrip('.', 1)[0]`
line 28: `assert py_file in py_files, "Did not create a corrosponding expected output for " + rock_file`